src/helpers/hotkeysManager.js

```diff
@@ -246,6 +255,26 @@ WebViewer(...)
           core.setCurrentPage(currPageNumber + 1);
         }
       },
+      /**
+       * Hold to switch to Pan mode and release to return to previous tool
+       * @name WebViewer.Hotkeys#Space
+       */
+      space: {
+        keyup: e => {
+          e.preventDefault();
+
+          setToolModeAndGroup(store, this.prevToolName);
+          this.prevToolName = null;
+        },
+        keydown: e => {
+          e.preventDefault();
```

Comment:
This prevents me from typing a space in a text box, for example a comment. We should make sure that no input elements are focused before triggering this. Or maybe check that the document is in focus instead